index.html

```diff
@@ -59,6 +59,128 @@ <h4 class="intro">Hey There. <br> I'm Kenny</h4>
+        <a href="#" class="btn2">View Project</a>
+        <div class="prof">
+            <h3 class="pro">First Portfolio <br> Dummy Project</h3>
+            <p class="pro">A simple design made by me, initial purpose was to house my future projects, and create a form for employers to fill when my service is needed. Just a dummy</p>
+            <ul class="pr">
+                <li>css</li>
+                <li>html</li>
+            </ul>
+        </div>
+        <a href="#" class="btn">View Project</a>
+        <div class="prof">
+            <h3 class="pro">First Portfolio <br> Dummy Project</h3>
+            <p class="pro">A simple design made by me, initial purpose was to house my future projects, and create a form for employers to fill when my service is needed. Just a dummy</p>
+            <ul class="pr">
+                <li>css</li>
+                <li>html</li>
+            </ul>
+        </div>
+        <a href="#" class="btn">View Project</a>
+        <div class="prof">
+            <h3 class="pro">First Portfolio <br> Dummy Project</h3>
+            <p class="pro">A simple design made by me, initial purpose was to house my future projects, and create a form for employers to fill when my service is needed. Just a dummy</p>
+            <ul class="pr">
+                <li>css</li>
+                <li>html</li>
+            </ul>
+        </div>
+        <a href="#" class="btn">View Project</a>
+        <div class="prof">
+            <h3 class="pro">First Portfolio <br> Dummy Project</h3>
+            <p class="pro">A simple design made by me, initial purpose was to house my future projects, and create a form for employers to fill when my service is needed. Just a dummy</p>
+            <ul class="pr">
+                <li>css</li>
+                <li>html</li>
+            </ul>
+        </div>
+        <a href="#" class="btn">View Project</a>
+        <div class="prof">
+            <h3 class="pro">First Portfolio <br> Dummy Project</h3>
+            <p class="pro">A simple design made by me, initial purpose was to house my future projects, and create a form for employers to fill when my service is needed. Just a dummy</p>
+            <ul class="pr">
+                <li>css</li>
+                <li>html</li>
+            </ul>
+        </div>
+        <a href="#" class="btn">View Project</a>
+    </section>
+    
+    <section class="about">
+        <h2 class="me">About me</h2>
+        <p class="info">Hello I’m a software developer! I can help you build any website with multi functions. You can look at some of my works to see some of my products and websites. If you like what I have and you have any project you need coded, hit  me up by dropping your information below.</p>
+    </section>
+    <div >
+        <ul class="but">
+            <li><a href="#" class="btn3">
+                My Resume</a></li>
+            <li><svg width="160" height="259" viewBox="0 0 160 259" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
As requested by the previous reviewer, kindly use the `backgroud-img` property to insert background images that are strictly for decoration purposes. This is the best practice for your document. You can use the `background-position` property to position it exactly where you want it.

Reply:
I have done that now